### ISSUE TITLE
missing YAML in  pip3 install git+<url>

### ISSUE THREAD

**Issue by rootVIII** [meanii/Search4#10]
Working on a fix for this... I may have just gotten it. Need to add a couple lines to setup.py...

Will get back in 20 minutes or so after some testing

**Comment by rootVIII** [comment]
Okay I figured it out I think... let me try to push the changes for setup.py

**Comment by rootVIII** [comment]
Gotta make your setup.py look like this:

```
from setuptools import setup

setup(
    name="search4",
    version="1.0.0",
    packages=["search4"],
    package_data={'search4': ['search4/*.yml']},
    include_package_data = True,
    url="https://github.com/7rillionaire/Search4",
    license="GPL-3.0",
    author="7rillionaire",
    description="A tool to search a particular username on almost every social platform and tell,"
    " whether the user with that username exists on that site or not.",
    entry_points={"console_scripts": ["search4=search4.search4:main"]},
    install_requires=['requests', 'colorama', 'pyyaml', 'jinja2'],
    data_files=[('search4', ['search4/search4.yml'])]
)

```

**Comment by ghost** [comment]
Fixed in #11!
